lms/static/js/groups/views/verified_track_settings_notification.js

```diff
@@ -0,0 +1,79 @@
+;(function (define) {
+    'use strict';
+    define(['jquery', 'underscore', 'backbone', 'gettext', 'edx-ui-toolkit/js/utils/string-utils',
+            'js/models/notification', 'js/views/notification'],
+        function ($, _, Backbone, gettext, StringUtils) {
+            /*global NotificationModel, NotificationView */
+
+            var VerifiedTrackSettingsNotificationView = Backbone.View.extend({
+
+                render: function () {
+                    // All rendering is done in validateSettings, which must be called with some additional information.
+                    return this;
+                },
+
+                validateSettings: function (isCohorted, cohortCollection, enableCohortsCheckbox) {
+                    if (this.model.get('enabled')) {
+                        var verifiedCohortName = this.model.get('verified_cohort_name');
+                        if (isCohorted) {
+                            var verifiedCohortExists = false;
+                            $.each(cohortCollection, function (_, cohort) {
+                                if (cohort.get('assignment_type') === 'manual' &&
+                                    cohort.get('name') === verifiedCohortName) {
+                                    verifiedCohortExists = true;
+                                    cohort.disableEditingName = true;
+                                }
+                                else {
+                                    cohort.disableEditingName = false;
+                                }
+                            });
+                            if (verifiedCohortExists) {
+                                this.showNotification({
+                                    type: 'confirmation',
+                                    title: StringUtils.interpolate(
+                                        gettext("This course is using automatic cohorting for verified track learners. Cohorts cannot be disabled, and the manual cohort named '{verifiedCohortName}' cannot be renamed. If you wish to change the configuration, contact your PM."), // jshint ignore:line
```

Comment:
"this course is using" -> "this course uses".
"You cannot disable cohorts, and you cannot rename the manual cohort named ```'{verifiedCohortName}'```. To change the configuration for verified track cohorts, contact your edX partner manager."